framework/composer.json

```diff
@@ -75,7 +75,8 @@
         "cebe/markdown": "~1.0.0 | ~1.1.0",
         "bower-asset/jquery": "3.2.*@stable | 3.1.*@stable | 2.2.*@stable | 2.1.*@stable | 1.11.*@stable | 1.12.*@stable",
         "bower-asset/punycode": "1.3.*",
-        "bower-asset/yii2-pjax": "~2.0.1"
+        "bower-asset/yii2-pjax": "~2.0.1",
+        "http-interop/http-middleware": "0.5"
```

Comment:
`http-interop/http-middleware` is abandoned at packagist.org. According to the upgrade note `http-interop/http-server-middleware` should be used instead.